tests/astro.ts

```diff
@@ -0,0 +1,11 @@
+import { runInRepo } from '../utils'
+import { RunOptions } from '../types'
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		repo: 'withastro/astro',
+		build: 'build',
```

Comment:
I added a `build:ci` command that skips the `tsc` check since TypeScript is getting in our way.

```suggestion
		build: 'build:ci',
```

Reply:
Blocked by https://github.com/withastro/astro/pull/2869